### PR TITLE
chart: add chart files for helm-install

### PR DESCRIPTION
**Pull request by xmh19936688**
#237

### CODE REVIEW

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 254 Deploy Test Success

**Comment by zhao-kun** [comment]
This helm chart is dedicated to the K8s ingress controller of the Easegress. As Easegress can be used in different scenarios, I think you'd better explicitly explain the intention in the chart (including use more clear names or add some comments description)  for avoiding confusing users.

**Comment by localvar** [comment]
Please rename the `chart` folder as it is too general and may be used for another purpose later.

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 254 Deploy Test Success

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 254 Deploy Test Success

File: helm-charts/ingress-controller/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v2
+name: easegress-controller
```
Comment (localvar) [inline helm-charts/ingress-controller/Chart.yaml:2]:
```suggestion
name: easegress-ingress-controller
```

File: helm-charts/ingress-controller/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# easegress ingress-controller chart
+
+Use for easegress ingress controller installation by helm on kubernetes.
+
+## usage
```
Comment (localvar) [inline helm-charts/ingress-controller/README.md:5]:
```suggestion
# Easegress ingress controller chart

Helm charts for install Easegress ingress controller on Kubernetes.

## Usage
```

**Review by localvar — APPROVED** [review]

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 254 Deploy Test Success

**Review by benja-wu — APPROVED** [review]
LGTM

**Pull request merged**